### PR TITLE
Add text indicating when edited original publication date will

### PR DESCRIPTION
**Pull request by cavgovdig**
## What
Add text indicating when edited original publication date will be used as public timestamp when manual is published.

## Why
Original functionality on old manual show page, accidentally omitted during migration to new design system.
safe_join() used to preserve html <br> tag instead of interpreting as string literal and rubocop doesn't permit concatenating with html tags.

## Visual
image

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

Follow [these steps](https://guides.rubyonrails.org/upgrading_ruby_on_rails.html) if you are doing a Rails upgrade.

### CODE REVIEW

File: spec/helpers/application_helper_spec.rb [hunk 1]
```diff
@@ -227,14 +227,27 @@ def current_user_can_publish? = false
...
       manual.publish_tasks = []
 
       rows = manual_metadata_rows(manual)
 
       expect(rows).not_to include(include(key: "Last published"))
     end
+
+    it "indicates when originally published time will be used as public timestamp" do
+      manual = FactoryBot.build_stubbed(:manual, use_originally_published_at_for_public_timestamp: true)
+      allow_any_instance_of(ApplicationHelper).to receive(:publication_task_state).and_return("test publication task state")
```
Comment (ryanb-gds) [inline spec/helpers/application_helper_spec.rb:241]: If we're going to mock this (would it be possible not to using timecop or something?) do we want to check that it received the first item from `publish_tasks` as a parameter when `publication_task_state` was called?

**Review by ryanb-gds — COMMENTED** [review]
Just want to check we're writing the best test we can be, under the circumstances. Appreciate it's a bit of an awkward edge case to handle, by the look of it

File: spec/helpers/application_helper_spec.rb (hunk 1 above)
Comment (ryanb-gds) [reply spec/helpers/application_helper_spec.rb:247]: Oh I was just wondering if you'd mocked the `publication_task_state` helper method because it was time sensitive, in which case we could use the real function and use Timecop to freeze time so we could make a predictable assertion without mocking the method. But that's probably not the case. If that doesn't clarify things, ping me and we can jump on a call. It's not particularly important anyway

**Review by ryanb-gds — APPROVED** [review]
LGTM

File: spec/helpers/application_helper_spec.rb (hunk 1 above)
Comment (mtaylorgds) [reply spec/helpers/application_helper_spec.rb:247]: Ah, no, it's more to avoid having to worry about the logic of what that method returns, to keep this test more "unity".
